python/examples/handler.py

```diff
@@ -56,20 +47,15 @@ def echo(request, response, proxy):
     print "echo"
     yield tornado.gen.sleep(1)
     # stream args right back to request side
-    response.argstreams = [
-        InMemStream(request.endpoint),
-        request.argstreams[1],
-        request.argstreams[2]
-    ]
+    response.set_header(request.get_header_s())
```

Comment:
`request.get_header_s()` feels akward. 

What about:

`request.headers` - a dict of all headers
`request.headers.get('X-Uber-Source')` - get the header with the `X-Uber-Source` key.